<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Left Joints</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="SHOULDER_L">
           <property name="text">
            <string>Left Shoulder</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ELBOW_L">
           <property name="text">
            <string>Left Elbow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="WRIST_L">
           <property name="text">
            <string>Left Wrist</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="HAND_L">
           <property name="text">
            <string>Left Hand</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="HIP_L">
           <property name="text">
            <string>Left Hip</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="KNEE_L">
           <property name="text">
            <string>Left Knee</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ANKLE_L">
           <property name="text">
            <string>Left Ankle</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FOOT_L">
           <property name="text">
            <string>Left Foot</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Scene_F" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="title">
            <string>Center Joints</string>
           </property>
           <widget class="QPushButton" name="CENTER">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Hip Center</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="SPINE">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Spine</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="COLLAR">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Collar</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="HEAD">
            <property name="geometry">
             <rect>
              <x>460</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Head</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="Scene" name="W_Scene">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="kinectkit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Kinect Kit</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <widget class="QSpinBox" name="spinBox">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>171</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="RUN">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Run Dataset</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>600</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Live Kinect</string>
            </property>
           </widget>
           <widget class="QPushButton" name="RESET">
            <property name="geometry">
             <rect>
              <x>350</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="LIVE">
            <property name="geometry">
             <rect>
              <x>460</x>
              <y>40</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Live</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="STOP">
            <property name="geometry">
             <rect>
              <x>460</x>
              <y>70</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>STOP</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_3">
            <property name="geometry">
             <rect>
              <x>350</x>
              <y>70</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Scale/T</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_4">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>70</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Filter</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QSlider" name="horizontalSlider">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>171</width>
              <height>22</height>
             </rect>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="pageStep">
             <number>11</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Right Joints</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="SHOULDER_R">
           <property name="text">
            <string>Right Shoulder</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ELBOW_R">
           <property name="text">
            <string>Right Elbow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="WRIST_R">
           <property name="text">
            <string>Right Wrist</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="HAND_R">
           <property name="text">
            <string>Right Hand</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="HIP_R">
           <property name="text">
            <string>Right Hip</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="KNEE_R">
           <property name="text">
            <string>Right Knee</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ANKLE_R">
           <property name="text">
            <string>Right Ankle</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FOOT_R">
           <property name="text">
            <string>Right Foot</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Scene</class>
   <extends>QFrame</extends>
   <header location="global">Scene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
